### ISSUE TITLE
Multi membership restrictions 

### ISSUE THREAD

**Issue by barrunning** [WWBN/AVideo#6296]
Is there a way to restrict users to one membership level?
Currently with 3 tiers a person could choose all three tiers.

**Comment by DanielnetoDotCom** [comment]
What will be the difference between those tiers?

**Comment by barrunning** [comment]
Access to different videos and services.
Tier 3 includes 1 and 2, tier 2 includes 1.

**Comment by DanielnetoDotCom** [comment]
different videos you can control it with user groups, so the subscription may work

access different services, you may need to be clear

**Comment by barrunning** [comment]
Yes the videos are in user groups with the tiers , subscription tied to user groups.
The issue is in being in tier 1 and being able to then select tier 2 and 3 or being in tier 3 and selecting 1 or 2 without stopping the previous membership.

**Comment by DanielnetoDotCom** [comment]
So you can create 3 usergroups:

UG1
UG2
UG3

and 3 subscription plans and add the user groups to the plans

SUB1 = UG1
SUB2 = UG1 + UG2
SUB3 = UG1 + UG2 + UG3

each subscription plan can have more than 1 user group

**Comment by barrunning** [comment]
Is there a way for the system to notify the person they have a subscription and they need to cancel current subscription before signing up for a different subscription?

**Comment by DanielnetoDotCom** [comment]
Yes, the user can signup for more than one subscription.

currently there is no way to restrict into a single subscription only
